### DISCUSSION TITLE
HttpsOnlyMode / HttpAllowlist -- Firefox 115.13.0esr

### DISCUSSION THREAD

**Discussion by mixfreak** [mozilla/policy-templates#1129] [General]
Hello,

We are using the Policy Templates v5.12 for our Firefox 115.13.0esr (64-bit).

Unfortunately, the new following functions are not working.

HttpsOnlyMode - Configure HTTPS-Only Mode.
HttpAllowlist - Configure origins that will not be upgraded to HTTPS.

These are also not listed under about:config.

Please help.

Thank you.

**Comment by mkaply** [comment]
These policies were not ported to the Firefox ESR. They are available in the 128 ESR which was released this week.

I realize this wasn't clear in the readme for the 127 policy release, but it was in the enterprise release notes:

https://support.mozilla.org/en-US/kb/firefox-enterprise-127-release-notes
